index.html

```diff
@@ -5,40 +5,88 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>My Portfolio skeleton</title>
-  <link rel="stylesheet" href="style.css">
-  <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet" />
-  <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet" />
+  <title>My Portfolio mobile sk</title>
 </head>
 
 <body>
   <main>
-      <header>
-         <nav class="navbar">
-        <div class="menu-ham">
-
-            <img src="images/icon-menu.svg" alt="ham icon menu"/>
-          </div>
+    <header>
+      <div id="logonav">
+        <h2 class="logo">
+          <a href="#">My Logo</a>
+        </h2>
+        <nav>
+          <ul>
+            <li>
+              <a href="#"><img class="menu" src="./images/IconMenu.svg" alt="ham"></a>
+            </li>
+          </ul>
         </nav>
-      </header>
-      
-      <section class="intro">
+      </div>
+    </header>
 
-        <div>
-          <h1 class="name">Charlie Giorgania</h1>
-          <p class="intro-text">
-            Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my
-            works.
-            If you like what you see and have a project you need coded, don’t hesitate and contact me.
-          </p>
+    <section class="intro">
+      <div id="back-images-intro">
+
+        <h1>I´m Andy<br>Glad to see<br>you!</h1>
+        <p class="intro-text">I’m a software developer! I can help you
+          build a product, feature or website. Look
+          through some of my work and
+          experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <a href="#">
+          <h3 class="connectlink">LET´S CONNECT</h3>
+        </a>
+
+        <div class="socialmedia">
+          <ul class="icons">
+            <li><button type="button"><img src="./images/twitter.svg" alt="twitter"></button></li>
+            <li><button type="button"><img src="./images/linkedin.svg" alt="linkedin"></button></li>
+            <li><button type="button"><img src="./images/m.svg" alt="M"></button></li>
+            <li><button type="button"><img src="./images/github.svg" alt="Git"></button></li>
+            <li><button type="button"><img src="./images/vector.svg" alt="V"></button></li>
+          </ul>
```

Comment:
[OPTIONAL] It seems the previous reviewers accidentally missed this requirement, therefore it is optional. However, if this was the first review, this would be a [REQUIRED] suggestion. I strongly encourage you to implement it.

According to the evaluation guidelines, the icons should be wrapped in a link tag.

<img width="608" alt="Screen Shot 2022-05-26 at 13 28 09" src="https://user-images.githubusercontent.com/82426682/170552659-0cb16e98-3a48-4f26-96ab-d82eeaa8b3f9.png">

Kindly fix this.